releasenotes/notes/process-agent-add-check-output-1a09261da182cd7e.yaml

```
@@ -0,0 +1,3 @@
+---
+enhancements:
+  - The flaire now includes output from the `processes` and `container` checks
```

I think we should add `process_discovery` here a well.
```suggestion
  - Agent flare now includes output from the `process` and `contain` checks
```

---

Ah thank you, apparently I had forgotten how to use spell check that day 😵‍💫

---

I will add process discovery to the flare as well!